layer/trace_layer.cpp

```diff
@@ -53,6 +53,29 @@ const VkLayerProperties kLayerProps = {
 
 const std::vector<VkExtensionProperties> kDeviceExtensionProps = { VkExtensionProperties{ "VK_EXT_tooling_info", 1 } };
 
+/// An alphabetical list of device extensions which we do not report upstream if
+/// other layers or ICDs expose them to us.
+const char* const kUnsupportedDeviceExtensions[] = {
```

Comment:
As a future consideration I think a searchable list of strings in C++ reads better for me as an `unordered_set` or `set` of `string` but I think this is fine for getting this in and closing the issue.

Reply:
Understood.